src/flexlogger/automation/_data_rate_level.py

```diff
@@ -2,11 +2,31 @@
 
 
 class DataRateLevel(Enum):
-    """An enumeration describing the possible data rate levels."""
+    """An enumeration describing the possible data rate levels that channels can be configured to use.
+    For additional information on configuring data rates, visit
+    https://www.ni.com/docs/en-US/bundle/flexlogger/page/configuring-data-rates.html
+    """
 
     SLOW = 1
+    """The slow analog data rate level in Hertz at which your DAQ device acquires data.
+    """
```

Comment:
[nit] The newline on these single-line docstrings isn't necessary. It can all be on a single line.

`"""The slow analog data rate level in Hertz at which your DAQ device acquires data."""`

Reply:
Fixed.